example-app/app/nav.tsx

```diff
@@ -41,28 +41,52 @@ export default function Nav() {
                 <a>Profile</a>
               </Link>
             </li>
+            <li>
+              <Link href="/edge-profile" legacyBehavior>
```

Comment:
Is `legacyBehavior` still needed here (and in other links)?

Reply:
I was just moving this over from the old example app, I could change them - but I'd rather do that in another PR